### PR TITLE
autorecon: add page

### PR DESCRIPTION
**Pull request by fazlearefin**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/linux/autorecon.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+> A multi-threaded network reconnaissance tool which performs automated enumeration of services.
+> Note: dependency packages need to be installed for `autorecon` to function properly.
+> More information: <https://github.com/Tib3rius/AutoRecon>.
+
+- Perform reconnaissance on target host(s) (detailed scan results will be dumped in `./results`):
+
+`sudo autorecon {{host_or_ip1,host_or_ip2,...}}`
+
+- Perform reconnaisance on [t]arget(s) from a file:
```
Comment (gutjuri) [inline pages/linux/autorecon.md:11]:
```suggestion
- Perform reconnaissance on [t]arget(s) from a file:
```

File: pages/linux/autorecon.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# autorecon
+
+> A multi-threaded network reconnaissance tool which performs automated enumeration of services.
+> Note: dependency packages need to be installed for `autorecon` to function properly.
```
Comment (gutjuri) [inline pages/linux/autorecon.md:4]: This is the case for all programs. Should we list the dependencies explicitly here?

**Review by gutjuri — APPROVED** [review]
Thanks, LGTM apart from two comments that I have :)

File: pages/linux/autorecon.md (hunk 2 above)
Comment (fazlearefin) [reply pages/linux/autorecon.md]: Removed the line.

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**